### PR TITLE
Fix missing platform specific runtimes and incorrect runtime versions

### PR DESCRIPTION
**Pull request by Styr1x**

### CODE REVIEW

**Comment by wolfcomp** [comment]
seeing how all of them use the same 
```cs
new NugetLockfile.Dependency()
{
    Resolved = RUNTIME_VERSION
}
``` 
couldnt it be an idea to make that static?
something in the line of 
```cs
private static NugetLockfile.Dependency RuntimeDependency = new NugetLockfile.Dependency()
{
    Resolved = "6.0.5"
}

...
await GetDependency("Microsoft.AspNetCore.App.Ref", RuntimeDependency, pkgFolder, client);
```

**Comment by Styr1x** [comment]
I did refactor that part and on top made the package fetching parallel, this is now much faster than before.

File: Plogon/BuildProcessor.cs [hunk 1]
```diff
@@ -149,19 +160,20 @@ async Task GetDependency(string name, NugetLockfile.Dependency dependency, Direc
...
-
-        foreach (var dependency in runtime.Value.Where(x => x.Value.Type != NugetLockfile.Dependency.DependencyType.Project))
+        foreach (var runtime in lockFileData.Runtimes)
         {
-            await GetDependency(dependency.Key, dependency.Value, pkgFolder, client);
+            Log.Information("Getting packages for runtime {Runtime}", runtime.Key);
+            
+            await Task.WhenAll(runtime.Value
+                .Where(x => x.Value.Type != NugetLockfile.Dependency.DependencyType.Project)
+                .Select(dependency => GetDependency(dependency.Key, dependency.Value, pkgFolder, new HttpClient())).ToList());
```
Comment (goaaats) [inline Plogon/BuildProcessor.cs]: Why are you not reusing the HttpClient? It'll keep the connection to the server open if you do

**Review by goaaats — APPROVED** [review]

File: Plogon/BuildProcessor.cs (hunk 1 above)
Comment (Styr1x) [reply Plogon/BuildProcessor.cs]: Was too lazy to look up threading guarantees when you GetAsync on the same client from different threads. If you know that its fine I'll add it back in a sec.

Comment (goaaats) [reply Plogon/BuildProcessor.cs]: should be good

**Review by goaaats — APPROVED** [review]

File: Plogon/BuildProcessor.cs (hunk 1 above)
Comment (Styr1x) [reply Plogon/BuildProcessor.cs]: Done.

**Pull request merged**